packages/amplify_auth_cognito_plugin/amplify_auth_cognito/pubspec.yaml

```diff
@@ -11,6 +11,8 @@ dependencies:
   flutter:
     sdk: flutter
   amplify_auth_plugin_interface: 0.3.2
+  amplify_auth_cognito_android: 0.0.1
```

Comment:
What's the versioning strategy for these?

Reply:
What really matters is that all of the platform packages (i.e.. android / ios) are using the same version of their underlying native libs, and that the same versions of platform packages are imported by their respective app-facing packages. 

That being said, it might be a little easier to keep these straight if we just launched the packages with a version consistent with all the others (e.g launch amplify_auth_cognito_android as 0.3.3). 

Does anyone think that will look strange?

Reply:
Hmm yeah, I see, since the native version may not be bumped each release. I think consistency still makes sense with the other packages.

Reply:
Yeah consistency isn't strictly necessary but might be useful for sanity.

Reply:
I bumped the initial versions to match the rest of the library.